javascript/ql/src/Security/CWE-770/ResourceExhaustion.ql

```diff
@@ -0,0 +1,20 @@
+/**
+ * @name Resource exhaustion
+ * @description Allocating objects or timers with user-controlled
```

Comment:
Shouldn't this be Allocating.... to?

Reply:
No. "Allocating an object" generally means that it is created, with the implicit meaning that you have allocated some memory for it. It does look a bit weird now that you ask about it, but that is how it is. You can see som background information on [wikipedia](https://en.wikipedia.org/wiki/C_dynamic_memory_allocation).